### ISSUE TITLE
error handling if browser does not allow access to databases

### ISSUE THREAD

**Issue by mojoaxel** [jensarps/IDBWrapper#78]
It is possible to disable "Allow websites caches and databases" e.g. in IE11. 

If databases are disabled `this.idb` is `null`. This should be handled in [the `openDB` function](https://github.com/jensarps/IDBWrapper/blob/develop/idbstore.js#L252).

**Comment by jensarps** [comment]
Agreed, a more meaningful error message, and the option to handle this is a good thing. Awesome that you did the implementation already – many thanks!
